setup.py

```diff
@@ -213,6 +213,7 @@ def check_file_at_path(path2file):
         else:
             omplinkflags = ['-fopenmp']
 
+    tbb_root = None
```

Comment:
Do not eliminate tbb by yourself. We know that Numba 0.49.1 and 0.50.1 does not work with TBB beta08. Just do not install tbb in your conda environment.

Reply:
I agree, I will revert it. Thanks for catching it.